gradle/DependencyUpdates.gradle.kts

```diff
@@ -31,15 +33,15 @@ enum class ReleaseType(private val level: Int) {
 
 object DependencyUpdates {
     private val stableKeywords = arrayOf("RELEASE", "FINAL", "GA")
-    private val releaseRegex = "^[0-9,.v-]+(-r)?$".toRegex(RegexOption.IGNORE_CASE)
+    private val releaseRegex = "^[\\d,.v-]+(-r)?$".toRegex(RegexOption.IGNORE_CASE)
     private val rcRegex = releaseKeywordRegex("rc")
     private val betaRegex = releaseKeywordRegex("beta")
     private val alphaRegex = releaseKeywordRegex("alpha")
     private val devRegex = releaseKeywordRegex("dev")
```

Comment:
Maybe we can remove `com.github.ben-manes.versions` plugin, use @renovate-bot instead.

Reply:
I've looked at Renovate before but didn't have the time to set it up 👍

Reply:
Just enable it and remove all `dependencyUpdates` usages, after that we can kill the `buildSrc`.